tests/drivers/sensor/icm42688/boards/native_posix.conf

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_EMUL=y
+CONFIG_SENSOR=y
+CONFIG_EMUL_ICM42688=y
```

Comment:
Wouldn't you need these configs no matter what board you ran this test on? Shouldn't they then be in `prj.conf`?